### ISSUE TITLE
Import pvsumm issue

### ISSUE THREAD

**Issue by vikash130795** [arunbalajeev/query-video-summary#5]
Am not able to import pvsumm, it's always showing module cofigparser is not found, how could i import it, should i need to give path in setup.py packages part?

**Comment by andhusa** [comment]
Hi, I’m also struggling with importing qvsumm. Looks like qvsumm needs an older environment of any kind. It might not support Python 3, as Python 3 accepts `import configparser` instead of `import ConfigParser`

**Comment by tingchihc** [comment]
So, how should I do to import Configparser? and when I change to configparse, I got the error message about "No Module named 'model'. "

How can I adjust this problem?
@andhusa

**Comment by andhusa** [comment]
I went down the same lane as you. ‘model’ is reffering to the file in the same folder. So to fix that you simply change `import model` to `from . import model`

Also change `import shells`  to `from . import shells`, as the same error occurs on that line.

I hope you can find a way to run it, because I could not find a solution as it was always a new error occuring. As i said, I think it is possible to run it on a downgraded Python.

@ting-chih

**Comment by tingchihc** [comment]
you mean this code should run in python2?
@andhusa

**Comment by andhusa** [comment]
Yes, it should not be necessary to change the written code. I just tried that quick, but I got an error on that I could not manage to fix at a quick look.

@ting-chih

**Comment by andhusa** [comment]
Did you manage to run it @ting-chih ?

**Comment by tingchihc** [comment]
@andhusa  NO
